domains/transfer-agreements/TransferAgreements.API/API/Data/ApplicationDbContext.cs

```diff
@@ -12,4 +12,5 @@ public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : ba
 
     public DbSet<TransferAgreement> TransferAgreements { get; set; }
     public DbSet<TransferAgreementHistoryEntry> TransferAgreementHistoryEntries { get; set; }
+    public DbSet<Connection> Connections { get; set; }
```

Comment:
I strongly suggest that you make the necessary C4 container diagrams and maybe component diagrams for the Transfer agreement domain as part of this PR.

Reply:
Sure - where do you suggest we put it?

Reply:
Christoffer did some work on the C4 diagrams, so ask him for a status. Otherwise, there is doc\architecture\domains\transfer-agreements.md in energy-origin.